app/models/claim/base_claim.rb

```diff
@@ -188,6 +188,7 @@ def assessment
     before_validation do
       errors.clear
       destroy_all_invalid_fee_types
+      reset_transfer_details unless case_transferred_from_another_court
```

Comment:
I would not call this that because there is a Claim::TransferDetail model that it could get confused with e.g. claim.transfer_detail

perhaps the rather unpleasantly long `transfer_court_details`